toolkit/smart-contracts/commands/src/governed_map.rs

```diff
@@ -85,6 +90,22 @@ pub struct RemoveCmd {
 	genesis_utxo: UtxoId,
 }
 
+#[derive(Clone, Debug, clap::Parser)]
+pub struct UpdateCmd {
+	#[clap(flatten)]
+	common_arguments: crate::CommonArguments,
+	#[arg(long)]
+	key: String,
+	#[arg(long)]
+	value: ByteString,
+	#[arg(long)]
+	current_value: Option<ByteString>,
+	#[clap(flatten)]
+	payment_key_file: PaymentFilePath,
+	#[arg(long, short('g'))]
```

Comment:
```suggestion
	#[arg(long, short('c'))]
```
Please run `./partner-chains-demo-node smart-contracts governance init --help`
We use `c` for it. We use `g` for governance authority parameter.